spec/bees.spec.js

```diff
@@ -5,6 +5,10 @@ const {nests} = require('../nests.json');
 describe('The bee JSON data', () => {
   for (const [id, bee] of Object.entries(bees)) {
     describe(id, () => {
+      it('Has only one common name, not two common names separated by "or"', () => {
+        expect(bee.name).not.toMatch(/\bor\b/);
```

Comment:
Can't this be done in the schema?

Reply:
Oh, I suppose, thanks!

Reply:
Hrm&hellip; on second thought, I'd prefer to not use the schema for that—negative regular expressions, like "does not contain the world 'or'", can get hard to write and read. Using `.not.toMatch()` is a lot nicer, even if Jasmine is a bit overkill job :slightly_smiling_face: